src/pages/steps/AdditionalInformationStep.tsx

```diff
@@ -209,6 +222,41 @@ const AdditionalInformationStep = ({
     variant,
   ]);
 
+  // @ts-expect-error
+  const contactInfo = getEventByIdQuery.data?.contactPoint;
+
+  // @ts-expect-error
+  const bookingInfo = getEventByIdQuery.data?.bookingInfo;
+
+  const getMergedContactAndBookingInfo = useDeepCompareMemoize<
+    () => MergedInfo | undefined
+  >(() => {
+    if (!contactInfo) return;
+    if (!bookingInfo) return contactInfo;
+
+    const emails = new Set(contactInfo['email']);
+    const urls = new Set(contactInfo['url']);
+    const phones = new Set(contactInfo['phone']);
+
+    if (bookingInfo['email']) {
+      emails.add(bookingInfo['email']);
+    }
+
+    if (bookingInfo['url']) {
+      urls.add(bookingInfo['url']);
+    }
+
+    if (bookingInfo['phone']) {
+      phones.add(bookingInfo['phone']);
+    }
```

Comment:
```suggestion
    const emails = new Set(contactInfo.email);
    const urls = new Set(contactInfo.url);
    const phones = new Set(contactInfo.phone);

    if (bookingInfo.email) {
      emails.add(bookingInfo.email);
    }

    if (bookingInfo.url) {
      urls.add(bookingInfo.url);
    }

    if (bookingInfo.phone) {
      phones.add(bookingInfo.phone);
    }
```
```suggestion
    const emails = new Set(contactInfo.email);
    const urls = new Set(contactInfo.url);
    const phones = new Set(contactInfo.phone);

    if (bookingInfo.email) {
      emails.add(bookingInfo.email);
    }

    if (bookingInfo.url) {
      urls.add(bookingInfo.url);
    }

    if (bookingInfo.phone) {
      phones.add(bookingInfo.phone);
    }
```

https://eslint.org/docs/latest/rules/dot-notation